ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -150,11 +157,15 @@ export default function CustomSpendingCap({
                 ? 'custom-spending-cap-input-value'
                 : 'custom-spending-cap'
             }
+            id={
+              value > (currentTokenBalance || error)
```

Comment:
Should this be:

```
(value > currentTokenBalance) || error
```
?

Reply:
That makes more sense to me also. 

`value > (currentTokenBalance || error)`  was in the component before, so I thought there is a reason for that.